### PR TITLE
Add page meta descriptions for SEO

### PR DESCRIPTION
**Pull request by zalhabash**

### CODE REVIEW

File: src/routes/(home)/+page.svelte [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <svelte:head>
+	<meta name="description" content="Join us and practice Kendo, a traditional Japanese martial art using swords. Open to all. No experience needed." />
```
Comment (nstringham) [inline src/routes/(home)/+page.svelte:2]: It makes more sense in my head to put the description after the title.

**Comment by nstringham** [comment]
My gut reaction is that there's something really wrong with these descriptions. After thinking about it and looking at some examples and guidance I think it's the use of first person that felt to strange to me. I think it's weird because it's unclear who the narrator is. Does us refer to Google if I see the description on a google search results? What if someone post a link on social media does "us' refer to the person who sent the link?

**Review by nstringham — DISMISSED** [review]

File: src/routes/faq/+page.svelte [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <svelte:head>
 	<title>FAQ | Kendo Club at the University of Michigan</title>
+	<meta
+		name="description"
```
Comment (nstringham) [inline src/routes/faq/+page.svelte:4]: Is this voice typical for descriptions? It feels a little weird for me but I’m not supper familiar with the conventions

File: src/routes/resources/what-is-rei/+page.svelte [hunk 3]
```diff
@@ -1,5 +1,9 @@
 <svelte:head>
 	<title>What is Rei? | Kendo Club at the University of Michigan</title>
+	<meta
+		name="description"
+		content="Read an enlightening perspective about rei, an important part of kendo philosophy ."
```
Comment (nstringham) [inline src/routes/resources/what-is-rei/+page.svelte:5]: Do we want to include the author? I know some of google’s examples included an author.

File: src/routes/about/+page.svelte [hunk 4]
```diff
@@ -1,5 +1,9 @@
 <svelte:head>
 	<title>About Us | Kendo Club at the University of Michigan</title>
+	<meta
+		name="description"
```
[the remark this answers is not in the document]
Comment (zalhabash) [reply src/routes/about/+page.svelte:4]: Resolved in f656f83ce90ef77a5affdbc888cdce59d4a127ee

File: src/routes/calendar/+page.svelte [hunk 5]
```diff
@@ -4,6 +4,10 @@
 
 <svelte:head>
 	<title>Calendar | Kendo Club at the University of Michigan</title>
+	<meta
+		name="description"
+		content="Find details about club practices and upcoming kendo events in the area. "
```
[the remark this answers is not in the document]
Comment (zalhabash) [reply src/routes/calendar/+page.svelte]: Resolved in f656f83ce90ef77a5affdbc888cdce59d4a127ee

File: src/routes/faq/+page.svelte (hunk 2 above)
Comment (zalhabash) [reply src/routes/faq/+page.svelte:4]: Resolved in 07baaf88cf3671467bb0b794dc22b87ecdfc9756

File: src/routes/resources/what-is-rei/+page.svelte (hunk 3 above)
Comment (zalhabash) [reply src/routes/resources/what-is-rei/+page.svelte]: Resolved in 6b51d18dfdea95912165a5d3b7043a537c756c83

**Review by nstringham — DISMISSED** [review]

**Review by nstringham — APPROVED** [review]

**Pull request merged**